File: nexed/php-novice/02-level/Faculteit-626da7cd2e2a-2db3a9f747b4/faculteit.php
<?php

echo "Van welk getal wil je de faculteit weten?";

$getal = intval(readline("Van welk getal wil je de faculteit weten? "));

$res = 1;

for ($i = 1; $i <= $getal; $i++) {
    $res *= $i;
}

echo $res . PHP_EOL;

?>

File: nexed/python-novice/03-level/Faculty-5aad6a42dbe8-9b0c2bd14065/faculteit.py
def faculteit(fac):
    i = 1

    for j in range(1, fac+1):
        i *= j

    return i

oorsprong = int(input("Van welke getal wil je de faculteit weten? "))

kut = faculteit(oorsprong)

print(kut)
